components/ActionsDropdown/ActionsDropdown.tsx

```diff
@@ -62,6 +69,13 @@ export const ActionsDropdown: React.FC<IActionsDropdownComponentProps> = withMoo
 			if (React.isValidElement(child) && (child.props as any).children) {
 				return React.cloneElement(child, { key: i, children: recurseChildren((child.props as any).children) } as any);
 			}
+			if (React.isValidElement<IBoxComponentProps>(child) && child.type === ActionsDropdownButtonClose) {
```

Comment:
this is unnecessary, we don't this kind of logic for the button as its not using the JSX concepts of the library. Basically it will always render and just fire the onClose when clicked.